src/status_im/ui/screens/profile/user/views.cljs

```diff
@@ -72,6 +72,10 @@
      [qr-code-viewer/qr-code-viewer {:style styles/qr-code}
       value (i18n/label :t/qr-code-public-key-hint) (str value)]]))
 
+(defview app-version []
+  (letsubs [web3-node-version [:get :web3-node-version]]
+    [react/text (str build/version " (" (or web3-node-version "N/A") ")")]))
```

Comment:
I would move this computation to subscription, as view should just render value delivered by subscription
https://github.com/Day8/re-frame/blob/master/docs/SubscriptionsCleanup.md